src/main/java/com/delhimunchery/feedback/domain/Option.java

```diff
@@ -1,39 +1,56 @@
 package com.delhimunchery.feedback.domain;
 
-import javax.persistence.Column;
+import java.sql.Date;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
-import javax.persistence.Table;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 @Entity
-@Table(name = "options")
 public class Option {
+
   @Id
   @GeneratedValue
-  @Column(name = "option_id")
-  private int optionId;
-  private int score;
-  private String name;
-  @Column(name = "image_url")
+  private long id;
+  private long score;
```

Comment:
What does score signify?

Reply:
The value of the particular option
Among the set of valid option it shows the ranking between them

Reply:
Then change it to ranking or sortOrder?

Reply:
Okay

Reply:
> Then change it to ranking or sortOrder?

Done!!